### DISCUSSION TITLE
Blazor server using .net 5.0 example.

### DISCUSSION THREAD

**Discussion by duanepfeiffer** [pnp/pnpcore#802] [Q&A]
Does anyone know if there is an example of using PnP Core with Blazor server app and .net 5?  I'm not getting anywhere.

I think I may have figured out my problem.  I wasn't including a token provider.  The Demo.AspNetCore example here was helpful.

**Comment by DesenEnterprises** [comment]
did you find a solution and were you successful at converting this to server?

Would you be willing to share?

**Reply by duanepfeiffer** [reply]
Sure.  Let me gather some files for you.

**Reply by duanepfeiffer** [reply]
Hope these will help
[PnP.CoreBlazorNet5.zip](https://github.com/pnp/pnpcore/files/8710736/PnP.CoreBlazorNet5.zip)
.

**Reply by DesenEnterprises** [reply]
Thank you, I'll take a look!

**Reply by DesenEnterprises** [reply]
I guess I'm not familiar enough with blazor and sharepoint to get this to work for me. It's returning null context when I use your service to try to get a pnpcontext.

And I can't seem to integrate this with the demo pages in my server app.

Can you share a bit more of your project files to help me understand how you're implementing and calling this service?
